src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,125 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonRow } from '../PersonRow/Person';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SortType } from '../../types/SortType';
+import {
+  getArrowClass,
+  getSortOrder,
+} from './helper';
+
+interface Props {
+  people: Person[],
+}
+const SORT = 'sort';
+const SORT_ORDER = 'order';
+
+export const PeopleTable: React.FC<Props> = (props) => {
+  const [searchParams] = useSearchParams();
+  const { people } = props;
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Name
+              <Link
+                to={{
+                  search: getSearchWith(searchParams, {
+                    [SORT]: SortType.Name,
+                    [SORT_ORDER]: getSortOrder(searchParams),
+                  }),
+                }}
+              >
+                <span className="icon">
+                  <i
+                    className={cn('fas',
+                      { [getArrowClass(searchParams, SortType.Name)]: true })}
+                  />
+                </span>
+              </Link>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Sex
+              <Link
+                to={{
+                  search: getSearchWith(searchParams, {
+                    [SORT]: SortType.Sex,
+                    [SORT_ORDER]: getSortOrder(searchParams),
+                  }),
+                }}
+              >
+                <span className="icon">
+                  <i
+                    className={cn('fas',
+                      { [getArrowClass(searchParams, SortType.Sex)]: true })}
+                  />
+                </span>
+              </Link>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Born
+              <Link
+                to={{
+                  search: getSearchWith(searchParams, {
+                    [SORT]: SortType.Born,
+                    [SORT_ORDER]: getSortOrder(searchParams),
+                  }),
+                }}
+              >
+                <span className="icon">
+                  <i
+                    className={cn('fas',
+                      { [getArrowClass(searchParams, SortType.Born)]: true })}
```

Comment:
If there is no specific condition for the class, it can be written as the second argument, it is not necessary to specify the `true` condition, fix it in the entire code